shell/platform/windows/platform_handler_unittests.cc

```diff
@@ -61,12 +64,13 @@ TEST(PlatformHandler, GettingTextCallsThrough) {
   TestBinaryMessenger messenger;
   TestPlatformHandler platform_handler(&messenger);
 
-  auto args = std::make_unique<rapidjson::Document>(rapidjson::kArrayType);
-  auto& allocator = args->GetAllocator();
-  args->PushBack(kTextPlainFormat, allocator);
-  auto encoded = JsonMethodCodec::GetInstance().EncodeMethodCall(
-      MethodCall<rapidjson::Document>(kGetClipboardDataMethod,
-                                      std::move(args)));
+  std::ostringstream jsonStringStream;
+  jsonStringStream << "{\"method\":\"" << kGetClipboardDataMethod
+                   << "\",\"args\":\"" << kTextPlainFormat << "\"}";
+  std::string jsonString = jsonStringStream.str();
+  unsigned char json[256];
```

Comment:
Sorry for my C++ rustiness, I just chose a big enough number here.  Can I set this to the actual size of the jsonStringStream somehow?

Reply:
If you know the size programmatically, you can use a `std::vector` and get at the pointer via `vec.data()` or `&vec[0]`. If you know the size and it can be hardcoded, then `std::array` and the same thing to get the pointer to the memory.

Reply:
Sorry was just looking at the two line github context. Since you've already got a std::string, why not use `jsonString.cstr()` to get at the `char*` and `jsonString.size()` where you need it?